app/assets/javascripts/discourse/tests/integration/widgets/post-menu-test.js

```diff
@@ -30,6 +31,11 @@ discourseModule(
           });
         });
       },
+
+      afterEach() {
+        resetPostMenuExtraButtons();
+      },
+
```

Comment:
I am curious if we can add afterEach to top level right after      setupRenderingTest(hooks);
like
```
hooks.afterEach
```

Reply:
O nice this works :)